projects/npm-tools/packages/npm-scripts/support/getFixture.js

```diff
@@ -4,16 +4,14 @@
  */
 
 const fs = require('fs');
-const path = require('path');
 const util = require('util');
 
+const getFixturePath = require('./getFixturePath');
+
 const readFile = util.promisify(fs.readFile);
 
 async function getFixture(fixture) {
-	return await readFile(
-		path.join(__dirname, '../__fixtures__', fixture),
-		'utf8'
-	);
+	return await readFile(getFixturePath(fixture), 'utf8');
```

Comment:
I know this was like this before, but you don't need `async` or `await` here. From the callers' perspective, these functions are identical promise-returning functions:

```js
async function getFixture1(fixture) {
    return await readFile(getFixturePath(fixture), 'utf8');
}

function getFixture2(fixture) {
    return readFile(getFixturePath(fixture), 'utf8');
}
```

And you can await on both in exactly the same way:

```js
await getFixture1('blah');
await getFixture2('blah');
```